src/app/components/control-bar/control-bar.component.html

```diff
@@ -18,6 +18,11 @@
         </div>
     </div>
     <div *ngIf="!loadingActiveProject && !loadingActiveProjectFailed" class="cell small-4 align-right align-middle grid-x">
+        <div *ngIf="canSwitchToPrivateMap" class="grid-x align-left align-middle">
```

Comment:
You can completely ignore this. 

Could this be moved next to the project name? 
I just thought that its purpose might be clearer if it was associated with the project.

Reply:
👍